apps/workers/sql-load/worker_test.go

```diff
@@ -138,13 +138,65 @@ func TestPrepareMeta(t *testing.T) {
 	trial.New(fn, cases).Test(t)
 }
 
+func TestMakeCsvHeader(t *testing.T) {
+	fn := func(in trial.Input) (interface{}, error) {
+		return MakeCsvHeader(in.Interface().([]byte), []rune(",")[0])
+	}
+	cases := trial.Cases{
+		"strings_header": {
+			Input:    []byte(`"first","second","third","fourth"`),
+			Expected: []string{"first", "second", "third", "fourth"},
+		},
+		"just_values_header": {
+			Input:    []byte(`first,second,third,fourth`),
+			Expected: []string{"first", "second", "third", "fourth"},
+		},
+		"mixed_values_header": {
+			Input:    []byte(`"first",second,"third",fourth`),
+			Expected: []string{"first", "second", "third", "fourth"},
+		},
+	}
+	trial.New(fn, cases).SubTest(t)
+}
+
+func TestMakeCsvRow(t *testing.T) {
+	schema := []DbColumn{
+		{Name: "id", FieldKey: "id"},
+		{Name: "name", FieldKey: "name", Nullable: true},
+		{Name: "count", FieldKey: "count", TypeName: "int", Default: trial.StringP("0")},
+		{Name: "percent", FieldKey: "percent", TypeName: "float", Nullable: true},
+		{Name: "num", FieldKey: "num", TypeName: "int", Nullable: true},
+	}
+
+	fn := func(in trial.Input) (interface{}, error) {
+		header := []string{"id", "name", "count", "percent", "num"}
+		return MakeCsvRow(schema, in.Interface().([]byte), header, []rune(",")[0])
```

Comment:
Tip: I find it easier to read the tests in the required type casting is done in the test func rather than in the case statements.
``` golang
fn := func(in trial.Input) (interface{}, error) {
		header := []string{"id", "name", "count", "percent", "num"}
		return MakeCsvRow(schema, []byte(in.Interface().(string)), header, []rune(",")[0])
}
cases := trial.Cases{
		"test_csv": {
			Input:    `"av1","myname",654321,0.145,123`,
			Expected: Row{"av1", "myname", int64(654321), 0.145, int64(123)},
		},
```